### ISSUE TITLE
Make use of Welcome Window configurable - enable direct start into Editor

### ISSUE THREAD

**Issue by Oliver-Loeffler** [gluonhq/scenebuilder#452]
This feature shall improve the usability of Scene Builder.

### Expected Behavior
The use of the Welcome Dialog shall be optional.
The Welcome Dialog provides access to recent files and to templates.
This is a very welcome feature at first Scene Builder start.

Depending on personal working style, direct start of Scene Builder to Editor Window can be desirable.
No functionality is lost as recent files and templates are accessible from file menu.

The welcome dialog should offer a toggle switch or a check box such as: `Show Welcome Screen at Start [x]`.
When checked, the Welcome Dialog shall be presented at program start.
When unchecked, Scene Builder shall directly start into the editor window with a new project.

If a new version of Scene Builder is installed, the Welcome Dialog dialog behavior should follow existing configuration.

### Current Behavior
Scene Builder always opens the welcome dialog window.

### Context
User Experience, UX

**Comment by AlmasB** [comment]
> When unchecked, Scene Builder shall directly start into the editor window with a new project.

or perhaps, open most recent project, similar to IntelliJ?

**Comment by Oliver-Loeffler** [comment]
Well, its also a matter of taste, so this can be also a setting in app preferences.

**Comment by AlmasB** [comment]
You are right, I think the preferences data should be discussed and figured out early. To improve UX we will provide various options and they need a place to live. Perhaps worth opening a discussion post.

As far as this issue is concerned, I'm happy to implement the following:

> The welcome dialog should offer a toggle switch or a check box such as: Show Welcome Screen at Start [x].
When checked, the Welcome Dialog shall be presented at program start.
When unchecked, Scene Builder shall directly start into the editor window with a new project.

**Comment by Oliver-Loeffler** [comment]
If you consider this change already for Scene Builder 18, a few additional settings will fit on the Preferences dialog.
But for Scene Builder 19 we definitely need a rework of this.

**Comment by AlmasB** [comment]
Ideally, I'd like to get this and #475 done before the next release. I assume SB follows javafx release cycles? Then we have a bit of time before 18.
